### DISCUSSION TITLE
ingress的annotations中 higress.io/destination如何生效

### DISCUSSION THREAD

**Discussion by Notailli** [higress-group/higress#358] [Q&A]
看代码是监听到ingress的创建，会去判断annotations是否有 higress.io/destination这个值，有的话，会生成DestinationRule配置，但是不会创建DestinationRule，是将DestinationRule直接生成XDS配置，下发给envoy了么？代码中在哪可以找到？

**Comment by johnlanni** [comment]
https://github.com/alibaba/higress/blob/main/pkg/ingress/kube/ingressv1/controller.go#LL993C22-L993C47

FYI
